src/tests/functional/shared_test_classes/src/single_layer/interpolate.cpp

```diff
@@ -168,4 +168,103 @@ void Interpolate1LayerTest::SetUp() {
     function = std::make_shared<ngraph::Function>(results, params, "interpolate");
 }
 
+namespace v11 {
+
+std::string InterpolateLayerTest::getTestCaseName(const testing::TestParamInfo<InterpolateLayerTestParams>& obj) {
+    InterpolateSpecificParams interpolateParams;
+    InferenceEngine::Precision netPrecision;
+    InferenceEngine::Precision inPrc, outPrc;
+    InferenceEngine::Layout inLayout, outLayout;
+    InferenceEngine::SizeVector inputShapes, targetShapes;
+    std::string targetDevice;
+    std::map<std::string, std::string> additional_config;
+    std::tie(interpolateParams, netPrecision, inPrc, outPrc, inLayout, outLayout, inputShapes, targetShapes, targetDevice, additional_config) = obj.param;
+    std::vector<size_t> padBegin, padEnd;
+    std::vector<int64_t> axes;
+    std::vector<float> scales;
+    bool antialias;
+    ngraph::op::v4::Interpolate::InterpolateMode mode;
```

Comment:
Using v4 for interpolate modes is intended instead of util::InterpolateBase?

Reply:
Updated all parameters to use values from `util::InterpolateBase`